src/main/kotlin/com/hibob/academy/dao/PetDao.kt

```diff
@@ -48,6 +44,13 @@ class PetDao @Inject constructor(private val sql: DSLContext) {
             .and(petTable.companyId.eq(companyId)))
             .fetch(petMapper)
 
+    fun getPetById(petId: Long, companyId: Long): PetData? =
+        sql.select(petTable.id, petTable.name, petTable.type, petTable.companyId, petTable.dateOfArrival, petTable.ownerId)
+            .from(petTable)
+            .where(petTable.id.eq(petId))
+            .and(petTable.companyId.eq(companyId))
+            .fetchOne(petMapper)
+
```

Comment:
This function related to this pr?

Reply:
it wasn't in the assignment but i think its a function that could by useful